lib/routes/oauth.js

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var validators = require('./validators')
+var HEX_STRING = validators.HEX_STRING
+
+module.exports = function (log, isA, error, db) {
+
+  var routes = [
+    {
+      method: 'POST',
+      path: '/oauth/sessions/revoke',
+      config: {
+        auth: {
+          strategy: 'oauthToken',
+          scope: ['account']
```

Comment:
We'll probably want to factor this out into a named constant eventually, but I don't think we're quite at the "nits and cleanups" stage of this PR yet :-)